diff --git a/main/nginx/APKBUILD b/main/nginx/APKBUILD
index b07cca0..fec8cb9 100644
--- a/main/nginx/APKBUILD
+++ b/main/nginx/APKBUILD
@@ -18,7 +18,7 @@ arch="all"
 license="custom"
 depends=""
 makedepends="linux-headers gd-dev geoip-dev libxml2-dev libxslt-dev
-	libressl-dev paxmark pcre-dev perl-dev pkgconf zlib-dev"
+	libressl-dev openldap-dev paxmark pcre-dev perl-dev pkgconf zlib-dev"
 checkdepends="perl perl-fcgi perl-io-socket-ssl perl-net-ssleay
 	perl-protocol-websocket uwsgi-python"
 pkgusers="nginx"
@@ -110,6 +110,7 @@ _add_module "http-upstream-fair" "v0.1.2" "https://github.com/itoffshore/nginx-u
 _add_module "rtmp" "v1.1.11" "https://github.com/arut/nginx-rtmp-module"
 _rtmp_provides="$pkgname-rtmp"  # for backward compatibility

+_add_module "http-auth-ldap" "master" "https://github.com/kvspb/nginx-auth-ldap"

 prepare() {
 	local file; for file in $source; do
@@ -288,4 +289,5 @@ a22cfab85f5a15cf4b778749227caee559982dbe7711e1c5698456b3821943ec66b7a980ed56612b
 c853b041cecc3521f888be3d0483c6cf23d239259ac4f5d35b10d4483b0fb06d1d22060adedcf6605b220d99f9051faf300d06989736d510b4486f943d3a675e  set-misc-nginx-module-0.31.tar.gz
 c31c46344d49704389722325a041b9cd170fa290acefe92cfc572c07f711cd3039de78f28df48ca7dcb79b2e4bbe442580aaaf4d92883fd3a14bf41d66dd9d8c  nginx-upload-progress-module-0.9.2.tar.gz
 4da7734301d21cd696fcc3aed1a496a93be15af373307487622c0a5920e79d9b580fd5836de7f9c0b60c01485021ba85afae1abb471e703c2d6e23c60ffe7d0a  nginx-upstream-fair-0.1.2.tar.gz
-e7c897265d1e93b06f7e46a653b113e24d2451e2112a7a6da415f130928437444a0346832fd9c10042397fea6120e4e44acc2bccf649ec30ca5bffbf985672e2  nginx-rtmp-module-1.1.11.tar.gz"
+e7c897265d1e93b06f7e46a653b113e24d2451e2112a7a6da415f130928437444a0346832fd9c10042397fea6120e4e44acc2bccf649ec30ca5bffbf985672e2  nginx-rtmp-module-1.1.11.tar.gz
+5ce1a528114fb2905136fad4e60a47d3c48be1427afefe2d453a770521fbb11f900a7bd9cbe5861fb0ddda0feaca3395f42bd8ee75de325bd8cdc34d32e77e3e  nginx-auth-ldap-master.tar.gz"
